src/main/java/org/betonquest/betonquest/menu/RPGMenu.java

```diff
@@ -181,8 +181,9 @@ public ReloadInformation reloadData() {
                 }
             }
         }
-        final ChatColor color = (info.result == ReloadResult.FULL_SUCCESS) ? ChatColor.GREEN : ChatColor.YELLOW;
-        LOG.info(color + "Reloaded " + info.loaded + color + " menus");
+//        final ChatColor color = (info.result == ReloadResult.FULL_SUCCESS) ? ChatColor.GREEN : ChatColor.YELLOW;
+//        LOG.info(color + "Reloaded " + info.loaded + color + " menus");
+        LOG.info("Reloaded " + info.loaded + " menus");
```

Comment:
LOG.info() should probably take a Component, is what I'm thinking?

Reply:
I am not sure what you mean here?

Reply:
The reason I say this, is because there is color formatting that you are trying to pass to the logger here. Look at the commented out code. If you pass a Kayori Component to the logger, you can include this color formatting information.

Reply:
As this is a part of the reload, it should only be logged to the console. And we do not support colors for the Console (or we dont want them). Where we do support colors is, when you have a Command Sender (in this case it would be a ConsoleCommandSender) and then you could send the colored message to that command sender. 

So in general if it goes to the normal log (file and console) there is no color at all. Only the (Console)CommandSender should support color codes. But this is nothing we want as CommandSender, So it is fine to remove the color codes here.

But you should also remove the commented out lines ;)

Reply:
The custom colors in the console log result in constant error logging in my console. If this needs to support either logging destination we would need to support colors in Kyori here. But if you're ok just removing the color formatting, I'm good with that too.

Reply:
We are ok with removing 👍